CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## 1.0.2 (TBD, 2020)
 * Enhancements
     * `do_shell()` now saves the return code of the command it runs in `self.last_result` for use in pyscripts
+    * Ctrl-C now stops a running text script instead of just the current script command
```

Comment:
I'd call this a bug fix instead of an enhancement

Reply:
ok